tests/unit/panther_analysis_tool/test_main.py

```diff
@@ -18,16 +18,25 @@ def setUp(self):
 
     def test_valid_json_policy_spec(self):
         for spec_filename, _, loaded_spec, _ in pat.load_analysis_specs('tests/fixtures'):
-            if spec_filename == 'example_policy.json':
+            if spec_filename.endswith('example_policy.json'):
```

Comment:
I don't think this was ever getting into this if statement since the filename includes the entire path